libsel4/sel4_plat_include/morello-qemu/sel4/plat/api/constants.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <sel4/config.h>
+
+#include <sel4/config.h>
```

Comment:
Why twice?

Reply:
Opps, thanks for noticing, should be fixed now